board/aio.md

```diff
@@ -3,22 +3,47 @@ AIO API
 
 The AIO API supports reading analog input pins that measure the analog voltage signal between 0 and a maximum voltage (usually 3.3 or 5 Volts), then do Analog-to-Digital Conversion (ADC) with a resolution of 10 or 12 bits on most boards, so that the result (pin value) is 0 to 1023 or 0 to 4095, inclusively.
 
-On some boards access to AIO may be asynchronous. This API uses synchronous read.
+On some boards access to AIO may be asynchronous. This API uses synchronous methods for allowing lean implementations.
 
-The API object
---------------
-AIO functionality is exposed by the [`AIO`](#aio) object that can be obtained by using the [`aio()`](./README.md/#aio) method of the [`Board` API](./README.md/#board). See also the [Web IDL](./webidl.md).
+<a name="apiobject"></a>
+### The AIO API object
```

Comment:
Maybe we should name this ADC since there's no output? We've called it AIO for a long time in ZJS but maybe it should change.